code/app/src/main/java/com/cmput301w21t06/crowdfly/Database/UserController.java

```diff
@@ -24,9 +27,9 @@
  */
 //db setters should process shit in the class too
 public class UserController {
-    private static CollectionReference userCollection = GodController.getDb().collection("users");
+    private static CollectionReference userCollection = GodController.getDb().collection("Users");
```

Comment:
Lets try to use the FirestorePath class for these

Reply:
Just so we dont have to go in and change it in multiple placing. Could lead to bugs.

Reply:
Updated in new commit